components/Carousel.js

```diff
@@ -21,47 +21,44 @@ class Carousel {
   }
 
   _rotate(direction) {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
-    carouselTrack.style.transition = "transform 0.6s ease-in-out";
+    this._carouselTrack.style.transition = "transform 0.6s ease-in-out";
     this._counter += direction;
-    carouselTrack.style.transform = 'translateX(' + ( -this._sizeOfSlide * this._numberOfSlidesPerPage * this._counter ) + 'px)';
+    this._carouselTrack.style.transform = 'translateX(' + ( -this._sizeOfSlide * this._numberOfSlidesPerPage * this._counter ) + 'px)';
     this._changeIndicator(direction);
   }
 
   _restart() {
-    const carouselTrack = this._carouselElement.querySelector('.carousel__track');
-    carouselTrack.style.transition = "transform 0.6s ease-in-out";
-    carouselTrack.style.transform = 'translateX(' + 0 + 'px)';
+    this._carouselTrack.style.transition = "transform 0.6s ease-in-out";
+    this._carouselTrack.style.transform = 'translateX(' + 0 + 'px)';
```

Comment:
I believe you can just write this as `'translateX(0px)';`.

Reply:
done